app/Community/Concerns/ActsAsCommunityMember.php

```diff
@@ -98,6 +98,14 @@ public function isBlocking(User $user): bool
         return $this->getRelationship($user) === UserRelationship::Blocked;
     }
 
+    /**
+     * Check if two users are mutually following each other.
+     */
```

Comment:
```suggestion
```
nit: Normally I'm in favor of as much documentation as possible, but this only describes verbatim the function implementation.

Reply:
Yeah, that's fine by me. I only had it since "friends" and "mutually following" are different, but you're right that the code speaks for itself.